velox/exec/fuzzer/tests/PrestoSqlTest.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+#include "velox/exec/fuzzer/ToSQLUtil.h"
+#include "velox/functions/prestosql/types/JsonType.h"
+#include "velox/functions/prestosql/types/TimestampWithTimeZoneType.h"
+
+namespace facebook::velox::exec::test {
+namespace {
+TEST(ToSQLUtilTest, toTypeSql) {
+  EXPECT_EQ(toTypeSql(BOOLEAN()), "BOOLEAN");
+  EXPECT_EQ(toTypeSql(TINYINT()), "TINYINT");
+  EXPECT_EQ(toTypeSql(SMALLINT()), "SMALLINT");
+  EXPECT_EQ(toTypeSql(INTEGER()), "INTEGER");
+  EXPECT_EQ(toTypeSql(BIGINT()), "BIGINT");
+  EXPECT_EQ(toTypeSql(REAL()), "REAL");
+  EXPECT_EQ(toTypeSql(DOUBLE()), "DOUBLE");
+  EXPECT_EQ(toTypeSql(VARCHAR()), "VARCHAR");
+  EXPECT_EQ(toTypeSql(VARBINARY()), "VARBINARY");
+  EXPECT_EQ(toTypeSql(TIMESTAMP()), "TIMESTAMP");
+  EXPECT_EQ(toTypeSql(DATE()), "DATE");
+  EXPECT_EQ(toTypeSql(ARRAY(BOOLEAN())), "ARRAY(BOOLEAN)");
+  EXPECT_EQ(toTypeSql(MAP(BOOLEAN(), INTEGER())), "MAP(BOOLEAN, INTEGER)");
+  EXPECT_EQ(
+      toTypeSql(ROW({{"a", BOOLEAN()}, {"b", INTEGER()}})),
```

Comment:
Please, try structs with names that have spaces or other non-alphanumeric characters.

Try unsupported types as well.